### DISCUSSION TITLE
Add automated letsencrypt certificate generation and renewal so that https is enabled by default

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [datalust/seq-tickets#2147] [Ideas]
Setting up https certificates is just another make-work task for sysadmins.  It would be great if seq could default to automatically generating certs if required using letsencrypt and provide https config overrides via env variables if users want to configure their own.

**Comment by nblumhardt** [comment]
Thanks for the suggestion, @rodneytamblyn 👍

**Comment by rodneytamblyn** [comment]
See also: https://github.com/rac021/certMe. Noting that there is a Docker image for auto generation of Letsencrypt certs (docker certbot)

R

**Comment by jbennink** [comment]
We have Seq behind traefik to get cert easily\
